modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase

```diff
@@ -277,6 +277,10 @@ definition {
 				productUnitPrice = 24,
 				shippingMethod = "Standard");
 
+			AssertElementNotPresent(
```

Comment:
This test does not have a description and the name is not clear about what happens in the test, here are my suggestions:
"This is a test for COMMERCE-10279. Users can verify that the list price is shown on the checkout summary rather than the promotion price that is higher than the list price."

Test name:
CanAssertListPriceRatherThanHigherPromotionPriceAtCheckout